siad/renter.go

```diff
@@ -25,6 +32,22 @@ func (d *daemon) renterDownloadHandler(w http.ResponseWriter, req *http.Request)
 	writeSuccess(w)
 }
 
+// renterFilesHandler handles the API call to list all of the files.
+func (d *daemon) renterFilesHandler(w http.ResponseWriter, req *http.Request) {
+	files := d.renter.FileList()
+	fileSet := make([]FileInfo, 0, len(files))
+	for _, file := range files {
+		fileSet = append(fileSet, FileInfo{
+			Available:     file.Available(),
+			Nickname:      file.Nickname(),
+			Repairing:     file.Repairing(),
+			TimeRemaining: int(file.TimeRemaining()),
```

Comment:
hmmm. okay.

Right now it also gets initialized improperly.

Do you want me to remove the conversion to an int?